apps/python/descriptions/scart.rst

```diff
@@ -69,17 +87,32 @@ Examples
       scart -I [file.mseed] [SDS archive]
       scart -I [file.mseed] --with-filecheck [SDS archive]
 
-#. Collect data from an FDSNWS server using the :ref:`global_recordstream`
-   interface and write to a miniSEED file. The data streams and the time spans are
-   defined in a list file using the option ``list``. The list can be generated e.g.
-   by :ref:`scevtstreams`.
+#. Collect data using the :ref:`global_recordstream` interface e.g. (FDSNWS server)
+   and write to a miniSEED file or import it into a local :term:`SDS` archive. The
+   data streams and the time spans can be defined in several ways. The list of option
+   ``list`` can be generated e.g. by :ref:`scevtstreams`.
```

Comment:
We actually also have an option directive: :option:`list`

Reply:
done